Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    it "processes value when @svn is not defined" do
```

Comment:
You shouldn't use instance variables to describe a test, since this doesn't really explain what it is supposed to do.